SDPi_Supplement/asciidoc/volume3/tf3-ch-8.2.8-biceps-content.adoc

```diff
@@ -4,11 +4,31 @@ NOTE:  This file contains the content that profiles BICEPS semantics for use in
 
 // 8.2.8
 [sdpi_offset=8]
-===== SDC/BICEPS semantic content
+==== SDC/BICEPS semantic content
 
 // 8.2.8.1
 ===== SDC/BICEPS Content Module
 
+====== BICEPS Extension Model
+
+IEEE 11073-10207 provides a powerful extension model.
+This enables manufacturers to add information that does not fit into the defined structure.
+Therefore, nearly every element in the participant and message model contains an extension point (`tns:Extension`).
+However, the fundamental rule is: Do not use an extension for anything that can be expressed using the given models.
+Especially, the powerful concept of coded values must be considered before using an extension.
+
+The boarders of using extension are given by:
+
+* Clause 8 of <<ieee_11073_10207_2017>>, especially R0109, and
```

Comment:
In other clauses, we denoted references by first mentioning the standard followed by the particular clause:

`<<ieee_11073_10207_2017>> Clause 8, especially R0129, [...]`

Reply:
```suggestion
* <<ieee_11073_10207_2017>> Clause 8, especially R0109, and
```